src/index.spec.ts

```diff
@@ -1,15 +1,12 @@
-import { OperatorType } from '@eppo/js-client-sdk-common/dist/dto/rule-dto';
-import { EppoValue } from '@eppo/js-client-sdk-common/dist/eppo_value';
```

Comment:
@aarsilv @sameerank I figured out the issue with this branch - I thought there was something about changed about typescript compilation but that wouldn't make sense since we are just changing the `jest` package.

The problem is that `OperatorType` and `EppoValue` are NOT exported by commons.

What I see in this test file is a lot of duplicated tests that are in commons - I've removed the ones that are referencing the private values and that leads me to a philosophical questions that only relevant tests to nodejs should be in this repo.